content/influxdb/v2.0/upgrade/v1-to-v2/docker.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Upgrade from InfluxDB 1.x to 2.0 with Docker
+description: >
+  Use the automated upgrade process built into the InfluxDB 2.x Docker image to
+  update InfluxDB 1.x Docker deployments to InfluxDB 2.x.
+menu:
+  influxdb_2_0:
+    parent: InfluxDB 1.x to 2.0
+    name: Upgrade with Docker
+weight: 101
+---
+
+Use the automated upgrade process built into the [InfluxDB 2.x Docker image](https://hub.docker.com/_/influxdb)
+to update InfluxDB 1.x Docker deployments to InfluxDB 2.x.
+
+- [Upgrade requirements](#upgrade-requirements)
+- [Upgrade examples](#upgrade-examples)
+
+{{% note %}}
+#### Export continuous queries before upgrading
+The automated upgrade process **does not** migrate InfluxDB 1.x continuous queries (CQs)
+to InfluxDB 2.x tasks. Export all of your CQs before upgrading to InfluxDB 2.x.
```

Comment:
```suggestion
to InfluxDB 2.x tasks (the 2.0 equivalent). Export all of your CQs before upgrading to InfluxDB 2.x.
```

Reply:
The upgrade process built into the container does export CQs to `/etc/influxdb2/v1-cq-export.txt`, not sure if we should mention that here as a fallback.